src/components/UniprotInput.vue

```diff
@@ -109,7 +109,17 @@ export default Vue.extend({
           }
         })
         .then(response => {
-          const parsedResponse = tsvParse(response.data)[0];
+          // The API query appears to match loosely on free text and yields
+          // multiple results. We care about exact identifier matches, so try to
+          // find one among the results.
+          const parsedResponse = tsvParse(response.data).find(
+            item => item["Entry"] === uniprotId
```

Comment:
[UniProtKB entries only use upper cases](https://www.uniprot.org/help/accession_numbers), so for increased usability you could allow case insensitive matches